### PR TITLE
chore(packages): migrate code to format - I1

### LINKED ISSUE #1: Initiation of Monorepo
Open discussion for the transition to a monorepo for our web/ui components.

## Current Plan
- [x] Move all UI components under new `web-components` repo (@dselman) — essentially done
- [x] Align version numbers / setup for Travis and npm deploy/publish (@jeromesimeon)
- [x] Do an unstable release to check that the packages can be consumed by users
  - [x] Including the new rollup packaging, etc
- [x] Decide when to officially publish
  - [x] Need to copy again the code from latest `master` in each UI component repos (@irmerk)
  - This could go nicely with the planned release of `markdown-editor` and `cicero-ui`
  - Instead of releasing, publish the first iteration of the new library
  - Point everyone to the new repo
- [x] Loose git history but migrate all issues under new repo (@irmerk)
- [x] Archive old repos

## Open Questions
- Component naming format? 
  - As is now?
  - New? e.g., `@accordproject/wc-cicero` `@accordproject/wc-concerto` `@accordproject/wc-markdown-editor`
  - Are we happy with `web-components`? A different name could possibly make for easier component naming
- What do we do with old demos, delete?
- Does it happen before or before `Slate57` components release?
- For versioning, does this mean we would start at `0.1.0` or continue on?
  - If we do not rename we have to increment
  - Highest is `concerto-ui` `0.80`+ so might be `0.90`
  - If new names we can do anything
- …?

### PR DESCRIPTION
**Pull request by irmerk**
# Closes #1 
Reformats the package structure for publishing

### Changes
- Changed the `package.json` dependencies (like `ContractEditor` for `MarkdownEditor`)
- Adjusted directory names
  - `@accordproject/markdown-editor => @accordproject/ui-markdown-editor`
  - `@accordproject/cicero-ui => @accordproject/ui-contract-editor`
  - `@accordproject/cicero-ui => @accordproject/ui-components`
  - `@accordproject/concerto-ui => @accordproject/ui-concerto`
  - `@accordproject/text-editor => @accordproject/ui-components`
- Updated `package.json` names
- Cleaned up dependencies 
- Adjusted rollup configs
- Adjusted `className` syntax

### Flags
- Need to be thorough and ensure this works seamlessly

### Author Checklist
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Extend the documentation, if necessary

### CODE REVIEW

**Comment by irmerk** [comment]
Will also need to change labels:
- Package: Cicero UI 💻 `=>` Package: Contract Editor 💻 
- Package: Concerto UI 💾 `=>` Package: UI Concerto 💾
- Package: Markdown Editor ⌨️ `=>` Package: Markdown Editor ⌨️
- Package: Text Editor 📝 `=>` Package: Components 🧩

File: packages/ui-components/rollup.config.js [hunk 1]
```diff
@@ -30,15 +29,22 @@ const postcssPlugins = [
...
 ];
 
-const config = outputs.map(({file, format}) => ({
+const config = outputs.map(({ file, format }) => ({
   input: 'src/lib/index.js',
   output: {
     file,
     format,
-    name: 'TextEditor',
+    name: 'CiceroUI',
```
Comment (jeromesimeon) [inline packages/ui-components/rollup.config.js:37]: Confused by this change. Why `CiceroUi` ?

Comment (irmerk) [reply packages/ui-components/rollup.config.js:37]: I must have had a fugue state when i made this change 🤷‍♀️

**Comment by irmerk** [comment]
Great points! I checked the Storybook and it looked fine... I'll go through and do another pass on the actual code and make sure it looks fine.

**Comment by irmerk** [comment]
Another FLAG is that the `package-lock.json` seems to still have older dependencies (`@accordproject/cicero-ui`) with a clean `lerna build`

File: README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ npm run storybook
 
 This repository is a monorepo, built using [lerna](https://lerna.js.org). Each package is published as an independent npm module.
 
-- [Markdown Editor](packages/markdown-editor): A WYSIWYG editor based on [Slate](https://www.slatejs.org) for markdown that conforms to the [CommonMark](https://spec.commonmark.org) specification.
-- [Text Editor](packages/text-editor): A basic text editor.
-- [Concerto UI](packages/concerto-ui): React components for models written in the [Concerto Modeling Language](https://github.com/accordproject/concerto).
-- [Cicero UI](packages/cicero-ui): Tools for implementing React components in your contract editing studio.
+- [Markdown Editor](packages/ui-markdown-editor): A WYSIWYG editor based on [Slate](https://www.slatejs.org) for markdown that conforms to the [CommonMark](https://spec.commonmark.org) specification.
+- [Contract Editor](packages/ui-contract-edtiro): An editor based on our [Markdown Editor](packages/ui-markdown-editor) for handling smart legal contracts.
```
Comment (DianaLease) [inline README.md:28]: typo `edtiro` -> `editor`

Comment (jeromesimeon) [inline README.md:28]: `edtiro -> editor`

**Pull request merged**